### PR TITLE
bugfix/fANOVA constant hyperparameters

### PR DESCRIPTION
**Pull request by sarah-segel**
- For fANOVA, remove constant hyperparameters from configspace (#9).

### CODE REVIEW

File: deepcave/plugins/hyperparameter/importances.py [hunk 1]
```diff
@@ -178,6 +179,26 @@ def process(run, inputs):
         if n_trees is None:
             raise RuntimeError("Please specify the number of trees.")
 
+        # Handle constant values in fANOVA: As the fANOVA implementation relies on pyrfr and pyrfr cannot be applied
+        # to constant hyperparameters (see https://github.com/automl/fanova/issues/81), as a workaround we remove
+        # constant hyperparameters before calculation.
+        # Note: This will break if there are conditions or forbiddens including constant hyperparameters.
```
Comment (alexandertornede) [inline deepcave/plugins/hyperparameter/importances.py:185]: Do we have that documented in the documentation? If not, please add that there as well.

**Review by alexandertornede — CHANGES_REQUESTED** [review]
Except for the documentation change, it looks good to me.

File: deepcave/plugins/hyperparameter/importances.py (hunk 1 above)
Comment (sarah-segel) [reply deepcave/plugins/hyperparameter/importances.py:184]: Added to the docs.

**Review by alexandertornede — APPROVED** [review]
LGTM

**Pull request merged**